packages/application-components/src/theming.ts

```diff
@@ -37,22 +37,28 @@ export const themesOverrides = {
     lineHeightForNavbarLink: uiKitDesignTokens.lineHeight20,
     marginBottomForPageTopBar: uiKitDesignTokens.spacing40,
     marginForCustomViewsSelectorAsTabular: `0 ${appKitSpacing55}`,
+    marginForCustomViewsSelectorAsTabularInCustomView: '0',
     marginForDialogContainerContents: `${uiKitDesignTokens.spacing30} 0 ${uiKitDesignTokens.spacing50} 0`,
     marginForModalPageHeader: `0 ${appKitSpacing55}`,
     marginForUserMenuItem: `${uiKitDesignTokens.spacing10} 0`,
     marginLeftForModalPageHeaderControls: uiKitDesignTokens.spacing50,
     marginRightForAppbar: appKitSpacing55,
     marginTopForDialogFooter: uiKitDesignTokens.spacing50,
     marginForPageContent: `${uiKitDesignTokens.spacing50} ${appKitSpacing55}`,
+    marginForPageContentInCustomView: `${uiKitDesignTokens.spacing50} 0`,
```

Comment:
As an alternative idea / approach, would it work if we override the css variables (theming) to change the margins instead of doing that explicitly in each component? 🤔

So basically in the CustomViewShell we inject styles in the in wrapper and override the css variables.

What do you think?

Reply:
Thanks @emmenko. That's a great idea 🔝 

The only drawback that I can see is that maybe it's a little more complicated to spot where things are happening in terms of maintanability.
With the new approach, the Page Layout components have a different UI when they are rendered within a Custom View Panel but it is not pretty straightforward why as the code which makes that happen is not in those components but in a new file (`custom-view-shell.styles.ts`).

I've opened [this PR](https://github.com/commercetools/merchant-center-application-kit/pull/3378) with the refactoring so we can all take a look at how it would look like before applying those changes here.

@commercetools/shield-team-fe it would be great to read your thoughts 🙏

Reply:
Thanks Carlos. I left a review in the other PR 🤗

Reply:
I merged the other PR into this one so we have your idea implemented here.